src/main/webapp/resources/js/pages/projects/share/ShareSuccess.jsx

```diff
@@ -81,19 +90,36 @@ function MultipleShared({ project, count }) {
  * @returns {JSX.Element}
  * @constructor
  */
-export function ShareSuccess({ removed, project, samples }) {
+export function ShareSuccess({ removed, project, samples, currentProject }) {
   const single = samples.length === 1;
 
+  const extra = [
+    <Button key="return" href={setBaseUrl(`/projects/${currentProject}`)}>
```

Comment:
Since setBaseUrl is used multiple times can you create a const for the setBaseUrl(`/projects/`) portion

Reply:
Removed in f6de4bd08a6253b2f51a7170afce67712ae2759c

Reply:
I see that you did this **const BASE_URL = setBaseUrl(`/project`);** Shouldn't it be **const BASE_URL = setBaseUrl(`/projects`);**?

Reply:
Well obviously.  Damn sorry.  Fixed in 11cba0999c957d4a5bfaa3a8c6c0d5b1efe8d273